vendor/k8s.io/kubernetes/pkg/kubectl/cmd/get/get.go

```diff
@@ -212,8 +212,13 @@ func (o *GetOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []stri
 		o.ServerPrint = false
 	}
 
+	templateArg := ""
```

Comment:
Update from https://github.com/kubernetes/kubernetes/pull/65377

Reply:
> Update from kubernetes/kubernetes#65377

@juanvallejo open another pull for me that I can squash in.